test/sql/order/test_order_variable_size_payload.test

```diff
@@ -5,8 +5,7 @@
 statement ok
 PRAGMA enable_verification
 
-# payload string that is longer than the 8MB initial string size - this tests whether RowDataCollection dynamically increases block capacity
-# TODO: test a mix of long and short strings
+# string that is longer than Storage::BLOCK_SIZE - this tests whether RowDataCollection dynamically increases block capacity
```

Comment:
Can we run these tests with force_external as well?

Reply:
Yes, good idea.